Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -742,15 +742,19 @@ protected function getRealpathCacheSize()
      * root directory. To make this command work for every case, read Composer's
      * vendor/ directory location directly from composer.json file.
      *
-     * @return string
+     * @return string Absolute path to vendor directory
      */
     private function getComposerVendorDir()
     {
-        $composerJson = json_decode(file_get_contents(__DIR__.'/../composer.json'));
-        if (isset($composerJson->config)) {
-            return $composerJson->config->{'vendor-dir'};
+        $composerVendorDir = "vendor";
+        $composerJson = json_decode(file_get_contents(__DIR__.'/../composer.json'),true);
+        if (isset($composerJson["config"]["vendor-dir"])) {
+            $composerVendorDir = $composerJson["config"]["vendor-dir"];
+            if (realpath($composerVendorDir) == $composerVendorDir) { //is absolute
```

Comment:
Nothing guarantees that the vendor dir configured in the `composer.json` file does not contain `../` when absolute.